docs/data/data-grid/filtering/CustomFilterPanelContent.js

```diff
@@ -65,9 +69,6 @@ export default function CustomFilterPanelContent() {
                 size: 'small',
                 sx: { mt: 'auto' },
               },
-              valueInputProps: {
-                required: true,
-              },
               deleteIconProps: {
                 sx: {
                   '& .MuiSvgIcon-root': { color: '#d32f2f' },
```

Comment:
Is there a way to change the Icon of your own? Maybe adding a prop would for the delete icon would be helpful

Reply:
That's out of the focus of this PR. I opened issue #4063 to add this icon to the slot of icons that can be override